src/delay.rs

```diff
@@ -47,17 +47,29 @@ impl DelayMs<u8> for Delay {
 
 impl DelayUs<u32> for Delay {
```

Comment:
Changes in here don't seem watchdog related

Reply:
Accidentally commited in the wrong branch (twice in a row)